python_anvil/api_resources/mutations/forge_submit.py

```diff
@@ -0,0 +1,127 @@
+from typing import Any, Dict, Optional, Text, Union
+
+from python_anvil.api_resources.mutations.base import BaseQuery
+from python_anvil.api_resources.mutations.helpers import get_payload_attrs
+from python_anvil.api_resources.payload import ForgeSubmitPayload
+
+
+DEFAULT_RESPONSE_QUERY = """
+{
+  id
+  eid
+  payloadValue
```

Comment:
Can you make this resolvedPayload? It's the most useful / complete of the payload resolvers.

Reply:
Also feel free to add status or any other things you are using in the examples in there